documents/motions/15.2022.1 - Suspensions and other Sanctions.md

```diff
@@ -1,45 +1,54 @@
+      1. Fines must only be issued if acts of the Regional Organization caused financial damage or financial loss to the WCA or any organization or individual within WCA Community.
+      2. The fine must not be larger than the actual damage or loss.
+   5. to withhold or withdraw prizes and honors (records, titles, medals, certificates) from a Registered Speedcuber.
+   6. to disqualify a Registered Speedcuber or specific results of a Registered Speedcuber during and/or after a WCA Competition.
+   7. to exclude a Community member from meetings and competitions of the WCA.
+2. The WCA Integrity Committee may exercise its suspensive rights under this Motion only if:
    1. a Registered Speedcuber has failed to pay fees for WCA Competitions, or
-   2. in the opinion of the WDC, the Registered Speedcuber is in breach of the Bylaws, Motions, or Regulations, or
-   3. the conduct of the Registered Speedcuber breaches or remains in breach of the Mission, Spirit, and/or Objectives of the WCA, or
-   4. the WDC considers that the Registered Speedcuber does not fulfil the requirements of eligibility for registration as a Registered Speedcuber.
-3. Before the WDC may exercise its suspensive rights under this Motion, the Registered Speedcuber must have been sent notice of the grounds and must have been given the opportunity to be heard by a WCA Delegate, the WCA Disciplinary Committee, or the WCA Board.
-4. A Registered Speedcuber who has been suspended by the WDC under this Motion may request for reinstatement by the WDC, with a motivation of the request.
-5. A Registered Speedcuber who has been suspended by the WDC under this Motion shall automatically cease to be suspended at the end of the fixed period, or once, in the opinion of the WDC, the set of circumstances specified has changed or ceases to exist.
-6. The WCA Ethics Committee shall have the following rights and duties, based on a Voting in the WCA Ethics Committee with a supermajority:
-   1. to issue fines against a Registered Speedcuber and suspend a Registered Speedcuber from participation in a WCA Competition until the fines are paid.
-      1. Fines must only be issued if acts of the Registered Speedcuber caused financial damage or financial loss to the WCA or any organization or individual within the WCA Community.
+   2. in the opinion of the WCA Integrity Committee, the Community member is in breach of the Spirit, Mission, Motions, Policies, Codes and/or Regulations of the WCA, or
+   3. the WCA Integrity Committee considers that the Registered Speedcuber does not fulfil the requirements of eligibility for registration as a Registered Speedcuber, or
+   4. the WCA Integrity Committee considers that the presence of a the Community member is a threat to the safety of other Community members or otherwise disruptive for the community.
```

Comment:
I would like to see a requirement or at least a should that the WIC consult with the relevant RO(s)?